specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,196 @@
+= Import cluster workflow
+
+The intent of import cluster workflow is to provide a mechanism to add a gluster storage cluster into tendrl for management.
+
+
+== Problem description
+
+This specification describes import cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of nodes and the clusters available for import to the Tendrl.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* A popup will be opened for it.
+
+* UI will filter the available nodes for importing the cluster.
+
+* The pop will list down the available nodes and a checkbox for enabling/disabling the cluster wide volume profiling.
+
+* Each node's name and IP address will be shown. 
```

Comment:
@cloudbehl I am fine with the older design for import cluster. @brainfunked @nthomas-redhat Whats your take on this.